code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -879,17 +883,18 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 	return tail_icon
 
 /mob/living/carbon/human/proc/set_tail_state(var/t_state)
-	var/image/tail_overlay = overlays_standing[TAIL_LAYER]
+	var/used_tail_layer = tail_alt ? TAIL_LAYER_ALT : TAIL_LAYER // VOREStation Edit - Alt Tail Layer
+	var/image/tail_overlay = overlays_standing[used_tail_layer] // VOREStation Edit - Alt Tail Layer
 	
-	remove_layer(TAIL_LAYER)
+	remove_layer(used_tail_layer) // VOREStation Edit - Alt Tail Layer

```

Comment:
Not really consistent with the other uses of alt layering. Might not actually work. Consider the following:
1) tail_alt is false, your tail is in TAIL_LAYER
2) You use the verb, tail_alt is inverted, and set_tail_state is called
3) set_tail_state then clears TAIL_LAYER_ALT out of overlays_standing and cuts it off the mob, leaving TAIL_LAYER, which remains populated with an image, and in your overlays, when really it should clear both, because: We're not using the old layer now, and we're about to update the new one so it needs to be gone

See belts and shoes, they both clear both potential layers at the top to avoid this case.

Reply:
@Leshana